var/spack/repos/builtin/packages/pmdk/package.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+import spack.architecture
+
+
+class Pmdk(Package):
+    """The Persistent Memory Development Kit (PMDK), formerly known as NVML,
+    is a library for using memory-mapped persistence, optimized specifically
+    for persistent memory
+    """
+
+    homepage = "http://pmem.io/pmdk/"
+    url      = "https://github.com/pmem/pmdk/archive/1.5.tar.gz"
+    git      = "https://github.com/pmem/pmdk.git"
+
+    version('develop', branch='master')
+    version('1.5',     sha256='6b069d7207febeb62440e89245e8b18fcdf40b6170d2ec2ef33c252ed16db2d4')
+    # in the nvml -> pmdk name change, these hashes needed to be updated
+    version('1.4.2',     sha256='df7e658e75d28cd80f6d2ff7b9fc9ae2885d52f8923fdbacecfd46215115fb4c')
+    version('1.4',       '8813455d9518b8d7e0c296a706314940')
+    version('1.3',         '32c41d0d7f1c855b7d2b9622631a6bc3')
+
+    # experimental rpmem
+    # depends_on('libfabric@1.4.2:')
+
+    depends_on('ncurses', when='@1.5:')
+
+    # documentation requires doxygen and a bunch of other depenedncies that
+    # were not working properly on our contianers
+    patch('0001-make-doc-building-explicit.patch')
+
+    patch('0002-remove-secure-getenv.patch', when='@1.4:')
```

Comment:
Is this patch available anywhere else? Or is the issue it resolves documented somewhere?

Reply:
PR updated so that package.py doesn't include old versions or their patches any more